### PR TITLE
LRDOCS-9165 Experiences localization and translation

### PR DESCRIPTION
**Pull request by jorge-campo**
This PR proposes changes to the 'Content Page Personalization' based on the [LRDOCS-9165](https://issues.liferay.com/browse/LRDOCS-9165) and [LPS-128893](https://issues.liferay.com/browse/LPS-128893).

To improve the reader experience, the original [Content Page Personalization](https://learn.liferay.com/dxp/latest/en/site-building/personalizing-site-experience/experience-personalization/content-page-personalization.html) has been split into two different articles:

- Content Page Personalization, which is a concept topic explaining Experiences and Personalization.
- Creating and Managing Experiences, which is a procedural topic explaining how to create and manage Experiences.

### CODE REVIEW

File: docs/dxp/latest/en/site-building/personalizing-site-experience/experience-personalization/content-page-personalization.md [hunk 1]
```diff
@@ -44,43 +44,19 @@ In this example:
...
 
-![You can add, edit, delete, or change priority for Experiences.](./content-page-personalization/images/04.png)
+```note::
+   When a user visits a Page and there is an Experience targeted to the user's Segment but not localized into the user's language, the user sees the Experience in the default language.
+```
 
-1. Go to *Site Administration* &rarr; *Site Builder* &rarr; *Pages*.
-2. Click the Actions (![Actions](../../../images/icon-actions.png)) menu and select *Edit*.
-3. Click on the Experience you want to manage. From here you can
+You can add or remove languages from the Experience using the [Edit Experience](./creating-and-managing-experiences.md#managing-content-page-experiences) (![Edit Experience](../../../images/icon-edit.png)) button, except for the default language that is always selected. The language selector button for the Experience only shows the languages configured for that particular Experience.
```
Comment (dgarciasarai) [inline docs/dxp/latest/en/site-building/personalizing-site-experience/experience-personalization/content-page-personalization.md:57]: Maybe we have to indicate something related about how to change the languages for the experiences? I mean, the available languages are the available ones for the site. Do we have an article related with that? It's just an idea 🤔

File: docs/dxp/latest/en/site-building/personalizing-site-experience/experience-personalization/creating-and-managing-experiences.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    ![Click on the current Experience to create a new one or select a different existing experience.](./content-page-personalization/images/01.png)
+
+4. Click on *New Experience*.
+5. Enter a name for the Experience and select the Segment for the audience you want to target, or [create a New Segment](../segmentation/creating-and-managing-user-segments.md) (available in Liferay DXP 7.2 Fix Pack 1+ and Liferay Portal 7.2 CE GA2+.)
+
+    ```note::
+       New Experiences are assigned to the *Anyone* Segment by default, and target all website visitors.
+    ```
+
+1. On the *New Experience* dialog, under the *Languages* section, select [additional languages](./content-page-personalization.md#managing-experience-localization) you want to target with the Experience. For example, if your default Content Page language is en-US and you want to provide a personalization for es-ES speakers, select this language as well.
```
Comment (dgarciasarai) [inline docs/dxp/latest/en/site-building/personalizing-site-experience/experience-personalization/creating-and-managing-experiences.md:22]: Here point 1 again? 🤔  Should we add from which version do you have this Languages section available?

**Review by dgarciasarai — COMMENTED** [review]
Reviewed! 😄 🚀 

I add a couple of comments, nothing blocking :)

Thanks! ❤️

File: docs/dxp/latest/en/site-building/personalizing-site-experience/experience-personalization/content-page-personalization.md (hunk 1 above)
Comment (jorge-campo) [reply docs/dxp/latest/en/site-building/personalizing-site-experience/experience-personalization/content-page-personalization.md:57]: That's a great idea, @dgarciasarai. I will clarify where the languages on that list come from and how to configure them. Indeed, we have an article about [Site Localization](https://learn.liferay.com/dxp/latest/en/site-building/site-settings/site-localization.html).

File: docs/dxp/latest/en/site-building/personalizing-site-experience/experience-personalization/creating-and-managing-experiences.md (hunk 2 above)
Comment (jorge-campo) [reply docs/dxp/latest/en/site-building/personalizing-site-experience/experience-personalization/creating-and-managing-experiences.md:22]: There is an admonition at the beginning of the article explaining that some Experiences functionality is only available in newer versions. But clarifying this point again won't hurt 😄  so I've included an additional note. Thanks!

**Comment by jorge-campo** [comment]
Thanks for your review, @dgarciasarai! 🚀

**Review by dgarciasarai — APPROVED** [review]

**Comment by jorge-campo** [comment]
Superseded by https://github.com/sez11a/liferay-learn/pull/425

**Pull request closed without merging**